### PR TITLE
Iz pack 5.x

### PR DESCRIPTION
**Pull request by fabmars**
- upgrade gradle to v6
- documentation & tests for IZPack v5 support (was already working, see #5)

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```
Comment (bmuschko) [inline gradle/wrapper/gradle-wrapper.properties]: This will likely force a higher compatible version of Gradle than necessary. Can you bump this down to at least a 5.x version?

File: build.gradle [hunk 2]
```diff
@@ -14,17 +17,17 @@ repositories {
 }
 
 dependencies {
-    compile localGroovy()
-    compile gradleApi()
-    testCompile('org.spockframework:spock-core:1.0-groovy-2.4') {
+    localGroovy()
+    gradleApi()
+    testImplementation('org.spockframework:spock-core:1.3-groovy-2.4') {
```
Comment (bmuschko) [inline build.gradle]: Let's make those changes independently please.

File: README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ using [IzPack](http://izpack.org/).
 
 To use the IzPack plugin, include in your build script:
 
-    apply plugin: 'com.bmuschko.izpack'
+    id 'com.bmuschko.izpack'
```
Comment (bmuschko) [inline README.md]: Should be wrapped in `plugins {}`.

File: src/main/groovy/com/bmuschko/gradle/izpack/CreateInstallerTask.groovy [hunk 4]
```diff
@@ -42,7 +42,6 @@ class CreateInstallerTask extends DefaultTask {
     File installFile
 
     @OutputFile
-    @PathSensitive(PathSensitivity.RELATIVE)
```
Comment (bmuschko) [inline src/main/groovy/com/bmuschko/gradle/izpack/CreateInstallerTask.groovy]: Why did you remove this? It's require for the build cache to work properly. Please revert.

Comment (fabmars) [reply src/main/groovy/com/bmuschko/gradle/izpack/CreateInstallerTask.groovy]: Because Gradle 6.x will not allow me. This confirms that I shoudln't have bumped that much. Let's do this again.
